locales/templates/app.pot

```diff
@@ -144,6 +144,23 @@ msgctxt "collection_separator_shots"
 msgid "%1$d shots"
 msgstr ""
 
+#. String displalyed when recognizing text on screenshot experiencing unknown
+#. error
+msgctxt "detail_ocr_error_failed"
+msgid "Recognize failed"
+msgstr ""
+
+#. String displayed when recognizing text on screenshots discovering the size
```

Comment:
The comment doesn't make a lot of sense to me (also typo on acceptable).

Does it mean that the OCR failed because the image is too big? Then `String displayed when text recognition fails because the image is too big` would be enough (ratio is a proportion).

Reply:
When the image is too big, it won't always cause the OCR failed. But it might have pretty low quality. However, the wrong or extreme proportion has a high possibility to have low OCR quality. As a result, we have a specific rule to define the "wrong or extreme proportion," and display the string for these cases. However, I agree to simplify the comment. "String displayed when text recognition fails because the image is too big" would be enough for context.